client/app/bundles/App/components/site.scss

```diff
@@ -0,0 +1,14 @@
+#root {
+  padding: 20px;
+}
+
+.markdown-body {
```

Comment:
Is there a plan to move the markdown-body and .site-nav styles to their respectable npm packages? We need to be wary of when we add styles to our doc site, as we could make consumers think that's how the components are styled in our repos.